content/articles/picking-pdf-and-image-from-phone-storage/index.md

```diff
@@ -0,0 +1,322 @@
+This tutorial is going to cover on how to open gallery and files apps from your android application and pick an image and a PDF file. You will learn about intents specifically implicit intents. 
```

Comment:
Change to "This tutorial will cover how to open gallery and files apps from your android application and pick an image and a PDF file. In addition, you will learn about intents, specifically implicit intents. "